bindings/rust/wasmedge-sdk/src/config.rs

```diff
@@ -1,14 +1,28 @@
 use super::wasmedge;
+use crate::error::ConfigError;
 
-#[derive(Debug, Default)]
+#[derive(Debug)]
 pub struct Config {
     pub(crate) inner: wasmedge::Config,
 }
-
 impl Config {
-    pub fn with_wasi() -> Self {
-        let config = Self::default();
-        let inner: wasmedge::Config = config.inner.enable_wasi();
-        Self { inner }
+    pub fn create() -> Result<Self, ConfigError> {
```

Comment:
If a user not put any additional modification to new an instance, s/he may like to use the `default` of config by Rust convention.  

We use create because the name is from C-Api, it may be nice to provide `default` function of `Default` trait to alias the create without parameters, or we can do it in different PR.

Reply:
Yes, Default is more convenient and friendly to Rust developers. We can consider introducing Default trait for such cases in future individual revisions.